scripts/build.sh

```diff
@@ -77,6 +77,7 @@ echo "COPY ${distributions}/*.zip"
 mkdir -p $OUTPUT/plugins
 cp ${distributions}/*.zip ./$OUTPUT/plugins
 
+./gradlew publishPluginZipPublicationToMavenLocal -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER -Dopensearch.version=$VERSION
 ./gradlew publishPluginZipPublicationToZipStagingRepository -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
```

Comment:
Why doesn't publishPluginZipPublicationToZipStagingRepository publish the artifacts to a place that it can be picked up by the other build?

Reply:
> Why doesn't publishPluginZipPublicationToZipStagingRepository publish the artifacts to a place that it can be picked up by the other build?

It does, but the other builds refer to artifacts - Maven repositories.